Bio/SeqFeature.py

```diff
@@ -1879,10 +1919,12 @@ def __str__(self):
         return out
 
     def _shift(self, offset):
+        """Legacy attribute to get extension (zero) as integer (PRIVATE)."""
```

Comment:
Copy and paste error? See the other ``_shift`` methods where you had a sensible docstring, e.g.

``Return a copy of the position object with its location shifted (PRIVATE).``